src/context_handle.rs

```diff
@@ -31,14 +74,36 @@ impl GContextHandle {
         initialize();
```

Comment:
initialize is not needed as it doesn't have `_t`

Reply:
Good point. I'll remove it everywhere.